prereise/gather/flexibilitydata/human_in_the_loop/demo/winter label.ipynb

```diff
@@ -0,0 +1,409 @@
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n",
+      "C:\\Users\\sienna\\AppData\\Local\\Temp\\ipykernel_26480\\3113503402.py:24: RuntimeWarning: divide by zero encountered in double_scalars\n",
+      "  delta = (flex_avg - other_avg)/flex_avg\n"
+     ]
+    }
+   ],
+   "source": [
+    "for file in os.listdir(winterFlexPath):\n",
+    "    \n",
+    "    temp = pd.read_csv(os.path.join(winterFlexPath, file))\n",
+    "    dates = temp.columns\n",
+    "    flex_label = np.array([])\n",
+    "    temperature_label = np.array([])\n",
+    "    price_label = np.array([])\n",
+    "    existing_days = []\n",
+    "    \n",
+    "    for i in temp.columns:\n",
+    "        \n",
+    "        if pd.to_datetime(i) not in summer2winter:\n",
+    "            continue\n",
+    "        else:\n",
+    "            existing_days.append(i)\n",
+    "            \n",
+    "        new_winter_day = pd.to_datetime(summer2winter[pd.to_datetime(i)])\n",
+    "        \n",
+    "        flex_avg = temp[i].iloc[15:20].copy()\n",
+    "        flex_avg = flex_avg.mean()\n",
+    "        \n",
+    "        other_avg = np.concatenate((temp[i].loc[0:14], temp[i].loc[20:])).mean()\n",
```

Comment:
It looks like index 14 isn't being accounted for in either average.

Reply:
You're right, this has been fixed.

Reply:
for file in os.listdir(winterFlexPath):
    
    temp = pd.read_csv(os.path.join(winterFlexPath, file))
    dates = temp.columns
    flex_label = np.array([])
    temperature_label = np.array([])
    price_label = np.array([])
    existing_days = []
    
    for i in temp.columns:
        
        if pd.to_datetime(i) not in summer2winter:
            continue
        else:
            existing_days.append(i)
            
        new_winter_day = pd.to_datetime(summer2winter[pd.to_datetime(i)])
        
        flex_avg = temp[i].iloc[15:20].copy()
        flex_avg = flex_avg.mean()
        
        other_avg = (temp[i].iloc[0:15]).mean()

Reply:
Done.